docs/feature_permissions.md

```diff
@@ -0,0 +1,46 @@
+# Feature Permissions
+
+> Written by Jade Keegan for the CSXL Web Application.<br> > _Last Updated: 11/3/2023_
```

Comment:
Super small nits: Prob do not need the `>` before last updated! Also date last updated may or may not be correct : )